File: src/main/java/FlyAway/flight/dto/FlightDetailsEmployeeDto.java
package FlyAway.flight.dto;

import FlyAway.flight.airport.dto.AirportBasicDto;

import java.time.LocalDateTime;
import java.util.UUID;

public record FlightDetailsEmployeeDto(
        UUID id,
        LocalDateTime departureDate,
        LocalDateTime arrivalDate,
        AirportBasicDto departureAirportDto,
        AirportBasicDto arrivalAirportDto
) {
}

File: frontend/src/app/services/models/flight-details-employee-dto.ts
/* tslint:disable */
/* eslint-disable */
import { AirportBasicDto } from '../models/airport-basic-dto';
export interface FlightDetailsEmployeeDto {
  arrivalAirportDto?: AirportBasicDto;
  arrivalDate?: string;
  departureAirportDto?: AirportBasicDto;
  departureDate?: string;
  id?: string;
}
